x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -185,11 +185,6 @@ export const allowedExperimentalValues = Object.freeze({
    */
   analyzerDatePickersAndSourcererDisabled: false,
 
-  /**
```

Comment:
switched to use `securitySolution.enableVisualizationsInFlyout` advanced setting (aka uiSettings)